src/main/java/com/olgaboiar/mint/HeaderGenerator.java

```diff
@@ -0,0 +1,14 @@
+package com.olgaboiar.mint;
+
+import static com.olgaboiar.mint.Constants.*;
+
+public class HeaderGenerator {
+
+
```

Comment:
👾

This is a nitpick since the extra line of whitespace here doesn't hurt the code, but by using a consistent style throughout the project and being deliberate about the style choices, small details like whitespace can have a positive impact on comprehension.

For example, using white space in a test case can help comprehend the subject under test.

```java
public class HeaderGeneratorTest {

    @Test
    public void testGeneratesCorrectStatusLine() {
        String statusCode = "200";

        HeaderGenerator generator = new HeaderGenerator();
        Header header = generator.generate(statusCode, "noop");

        assertEquals("HTTP/1.1 200 OK", header.getStatusLine());
    }
}
```